docs/backend/configuration.md

```diff
@@ -4,10 +4,11 @@ title: Configuration
 
 # Configuration
 
-We currently use two gems for configuring the application:
+We currently use three gems for configuring the application:
```

Comment:
I'd remove the number so that we don't need to change it if we increase or decrease the number of gems :D 

"We currently use a few gems"

Reply:
"We currently use the following gems..." :)